src/api/controllers/lines.js

```diff
@@ -45,6 +45,10 @@ function createLine(req, res) {
         res.status(401).json({
             "message": response.unauthorized
         });
+    } else if (req.user_type !== 'student') {
```

Comment:
needs to be req.user.user_type

user_type is a property of user which comes from the JWT in the request.
But user_type is not a direct attribute of the JWT, so this check rejects everybody, even students, as it is written right now

I tested it as is and neither recruiters nor students could create lines, for the above reason

Reply:
huh that's funny, i thought it worked the first time around, but maybe my eyes were deceiving me... yes, @kopaka7 please fix using `git commit --amend` and `git push origin issue-77-verify-student-for-post-lines -f`